dev/com.ibm.ws.container.service.compat/src/com/ibm/ws/util/BoundedBuffer.java

```diff
@@ -416,6 +434,7 @@ public BoundedBuffer(int capacity) throws IllegalArgumentException {
             getQueueLocks_[i] = new GetQueueLock();
         }
 
+        CpuInfo.addAvailableProcessorsListener(this);
```

Comment:
Should this be conditional on if the spin take property is not set?  i.e. if (!spinsTakeProp)